### DISCUSSION TITLE
error: maven-compiler-plugin not found

### DISCUSSION THREAD

**Discussion by sazcoaga** [dds-jv/foro#188] [Consultas de herramientas] [answered]
Buenas! estuve siguien los pasos que dijeron en el seminario para empezar a usar maven. El problema es que en el pom me sigue diciendo que no encuentra  la version de maven y me lo pone en rojo y ya busque en internet y todo pero no se como arreglarlo. 
Se supone que yo tengo la 3.8.1 pero probe tambien cambiando eso y sigue en rojo. Tambien probe lo de "invalidate cache" y probe tambien poner manualmente en settings -> maven la direccion del directorio donde tengo todos los archivos maven. 

![WhatsApp Image 2021-04-20 at 15 21 33](https://user-images.githubusercontent.com/62355034/115445375-2597ef00-a1ec-11eb-9515-9cb72d4b4af3.jpeg)

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola!

> El problema es que en el pom me sigue diciendo que no encuentra la version de maven y me lo pone en rojo 

Ojo, lo que te marca en rojo no habla de _maven_ sino del _maven-compiler-plugin_. Son dos cosas diferentes: el primero es la herramienta que usamos para construir los proyectos Java, mientras que el segundo es un componente particular que sirve para compilar. No tienen por qué coincidir en sus versiones (por ejemplo, yo tengo maven 3.3.9 pero estoy usando la versión 3.8.1 del compiler-plugin)

Por otro no me queda del todo claro si construiste ese pom a mano o utilizaste [el projecto base](https://github.com/dds-utn/java-base-project) que ofrecemos. Está bien en cualquiera de los casos, pero te lo pregunto para entender sobre que estamos trabajando. 

> Se supone que yo tengo la 3.8.1 pero probe tambien cambiando eso y sigue en rojo. 

Esto me genera dudas: la captura de pantalla muestra otra versión (deberías utilizar 3.8.1 que es la más nueva) y además el rojo lo estamos viendo a través de IntelliJ, ¿no? Un error del IDE no significa necesariamente que las cosas no anden, ¿probaste ejecutar `mvn test` desde la consola? ¿Qué arroja?

En cualquier caso, el problema debería resolverse ejecutando `mvn install` desde una consola en el directorio del proyecto. Probá hacer eso y decinos qué resultado arroja. 

¡Saludos!

**Reply by gastonprieto** [reply]
te sirvio @sazcoaga ?

**Reply by sazcoaga** [reply]
Hola! Perdon que recien veo la notificacion. Un compañero me ayudo a arreglarlo agregando otra linea y todo anda ahora. Pero se me hace un poco dificil explicar la situacion via escrito. 
Muchas gracias por la ayuda!
